plasmapy/physics/transport.py

```diff
@@ -27,8 +27,8 @@
 def Coulomb_logarithm(T,
                       n_e,
                       particles,
-                      z_mean=None,
-                      V=None,
+                      z_mean=np.nan*u.dimensionless_unscaled,
```

Comment:
The more I think about it, the more I come to the conclusion that we really should be taking advantage of annotations... Nothing to change here, just thinking out loud.

Well...

I have a loud keyboard, okay?